pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -17,8 +14,13 @@ import (
 	c "github.com/docker/swarm/pkg/multiTenancyPlugins/utils"
 	"github.com/gorilla/mux"
 	"github.com/samalba/dockerclient"
+	"io/ioutil"
+	"net/http"
+	"strings"
 )
 
+const NOTAUTHORIZED_ERROR = "Not authorized error. Tenant does not own container %s."
```

Comment:
I think the message  should be something along the lines of "No such container or that the user is not authorized for this container"